tests/core/test_model.py

```diff
@@ -21,6 +21,13 @@
 )
 from intelligence_layer.core.model import _cached_context_size, _cached_tokenizer
 
+INSTRUCTION = "Who likes pizza?"
+INPUT = "Marc and Jessica had pizza together. However, Marc hated it. He only agreed to the date because Jessica likes pizza so much."
+INSTRUCTION_PROMPT_TEMPLATE_PREFIX_LEN = (
+    59  # Length of "prefix" of Llama3-Instruct prompt template
+)
+INPUT_PREFIX_LEN = 2  # Additional new lines after instruction
```

Comment:
Generally agreed. For now we will keep the real models because we also have tests that call the actual endpoints that need them. We could think about adding some test-models in a central place for the more simple test cases.